src/web/app.py

```
@@ -272,7 +289,8 @@ def _combine_citations(podcast_results: dict, web_results: dict) -> List[dict]:
 async def generate_streaming_response(
     query: str,
     session_id: str,
-    history: Optional[List[dict]] = None  # TODO: Integrate with ADK session context
+    history: Optional[List[dict]] = None,  # TODO: Integrate with ADK session context
```

Ruff (0.14.8)

Unused function argument: `history`

(ARG001)